bee-api/bee-rest-api/src/endpoints/routes/api/plugins/debug/white_flag.rs

```diff
@@ -159,36 +120,28 @@ pub(crate) async fn white_flag<B: StorageBackend>(
         }
     }
 
-    // TODO
     let mut metadata = WhiteFlagMetadata::new(index, 0);
 
     // Wait for either all parents to get solid or the timeout to expire.
```

Comment:
```suggestion
    // Wait for all parents to get solid or the timeout to expire.
```
The `either` confused me :trollface: . At first I thought it would be associated with `parents` as in: `either parent to get solid`, but then realized it's `either ... or`. Removing it should be fine and less confusing to readers :slightly_smiling_face: